integration/src/test/java/org/apache/iotdb/db/integration/IoTDBLastIT.java

```diff
@@ -576,6 +577,128 @@ public void lastWithFilterTest() {
     }
   }
 
+  // test cases that empty container would be created
+  @Test
+  public void lastWithEmptyContainer() {
+    String[] retArray =
+        new String[] {
+          "500,root.ln.wf01.wt02.temperature,15.7,DOUBLE",
+          "500,root.ln.wf01.wt02.status,false,BOOLEAN",
+          "500,root.ln.wf01.wt02.id,9,INT32",
+          "600,root.ln.wf01.wt02.temperature,10.2,DOUBLE",
+          "600,root.ln.wf01.wt02.status,false,BOOLEAN",
+          "600,root.ln.wf01.wt02.id,6,INT32"
+        };
+
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+
+      PartialPath path = new PartialPath("root.ln.wf01.wt02.temperature");
+      IoTDB.metaManager.resetLastCache(path);
+
+      IMeasurementMNode node;
+      node = IoTDB.metaManager.getMeasurementMNode(path);
```

Comment:
It's not an IT way, in IT tests, we only care about whether the external results are correct, we can never get the inner state of IoTDB which should be put into UT.